openlibrary/catalog/marc/parse.py

```diff
@@ -23,26 +23,26 @@ class SeeAlsoAsTitle(MarcException):
 
 want = [
     '001',
-    '003', # for OCLC
-    '008', # publish date, country and language
-    '010', # lccn
-    '020', # isbn
-    '035', # oclc
-    '050', # lc classification
-    '082', # dewey
-    '100', '110', '111', # authors
+    '003',  # for OCLC
+    '008',  # publish date, country and language
+    '010',  # lccn
+    '020',  # isbn
+    '035',  # oclc
+    '050',  # lc classification
+    '082',  # dewey
+    '100', '110', '111',  # authors
     '130', '240', # work title
-    '245', # title
-    '250', # edition
-    '260', # publisher
-    '300', # pagination
-    '440', '490', '830' # series
-    ] + [str(i) for i in range(500,595)] + [ # notes + toc + description
-    #'600', '610', '611', '630', '648', '650', '651', '662', # subjects
-    '700', '710', '711', '720', # contributions
-    '246', '730', '740', # other titles
-    '852', # location
-    '856'] # URL
+    '245',  # title
+    '250',  # edition
+    '260',  # publisher
+    '300',  # pagination
+    '440', '490', '830'  # series
+    ] + [str(i) for i in range(500, 588)] + [  # notes + toc + description
```

Comment:
@cdrini this range has changed to exclude 59X fields which are local notes, and 589 which is not officially defined either.